spec/controllers/charts_controller_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Account::ChartsController, type: :controller do
+  describe "GET requests to actions" do
```

Comment:
there is no such component, this is too general description

just wrap each metod into separate describe
also tests are a bit poor.. mb add some data validity expectations?

Reply:
it's also OK. you can remove this describe at all :)